config/config_data_loader.ini

```diff
@@ -0,0 +1,22 @@
+[camera]
+interface=DataLoader
+
+[data_path]
+path_to_cloud=/home/anindex/data/clouds
+cloud_extension=.pcd
+path_to_rgb=/home/anindex/data/images
+rgb_extension=.png
+path_to_depth=/home/anindex/data/depths
+depth_extension=.png
+
```

Comment:
one last request:). could you please make these paths relative to the robosherlock locations. Do a check: if the relative path does not exist check it as an absolute path, if that is also invalid, throw an error and terminatr with a message.